app/filtering.js

```diff
@@ -364,6 +364,17 @@ function registerForHeadersReceived (session, partition) {
       muonCb({ cancel: true })
       return
     }
+
+    let parsedTargetUrl = urlParse(details.url || '')
+    let parsedFirstPartyUrl = urlParse(firstPartyUrl)
+    let trackingHeaders = ['Strict-Transport-Security', 'Expect-CT', 'Public-Key-Pins']
```

Comment:
these should be `const` not `let`

Reply:
minor: rename `trackingHeaders` to something like `trackableSecurityHeaders` so it's more clear why we are deleting them

Reply:
i think these should also include https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Public-Key-Pins-Report-Only